leakcanary-android-sample/src/release/java/com/example/leakcanary/ReleaseExampleApplication.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.leakcanary
+
+import android.util.Log
+import leakcanary.ConditionalHeapAnalyzer
+import leakcanary.ConditionalHeapAnalyzer.Result
+import shark.SharkLog
+import shark.SharkLog.Logger
+
+class ReleaseExampleApplication : ExampleApplication() {
+
+  override fun onCreate() {
+    super.onCreate()
+
+    // Useful to debug in release builds. Don't use in real builds.
+    SharkLog.logger = object : Logger {
+      override fun d(message: String) {
+        if (message.length < 4000) {
+          Log.d("LeakCanary", message)
+        } else {
+          message.lines().forEach { line ->
+            Log.d("LeakCanary", line)
+          }
```

Comment:
Why not build this into the standard logger?

Reply:
SharkLog is in a java only module.

Also, this should most definitely not be the default behavior, we generally don't want to log in prod (except here because we're testing it)